.gitignore

```diff
@@ -34,3 +34,4 @@ yarn-error.log*
 .npm/
 .cache/
 .yarnrc
+.env
```

Comment:
I would actually agree that this is a good thing to put in `.gitignore` but it breaks precedent, so for now it's probably better to revert this change.

Reply:
Okay okay

Reply:
Done